amivapi/users/__init__.py

```diff
@@ -32,3 +32,8 @@ def init_app(app):
     app.on_updated_users += project_password_status_on_updated
     app.on_replace_user += hash_on_update
     app.on_replaced_user += project_password_status_on_updated
+
+    # on_post_METHOD, triggered before sending the response by Eve
+    for method in ['GET', 'POST', 'PATCH', 'PUT']:
+        event = getattr(app, 'on_post_' + method + '_users')
+        event += hide_fields
```

Comment:
Regarding the current travis error:

Your new hooks should work, but the new implementation of `hide_fields` crashes the old hooks, which are still present.

You need to remove these lines above:
```
app.on_fetched_item_users += hide_fields
app.on_fetched_resource_users += hide_fields
```
(Deprecated anyway since covered by the `on_post` hook)

Then everything should work.

Reply:
Unfortunately, there are still errors with the Travis build.

Reply:
I will investigate.

In the mean time I have noticed something else: You can remove the `PUT` from methods. AMIV API does not support `PUT~

Reply:
Are you sure?

The following line indicates something else:
https://github.com/amiv-eth/amivapi/blob/149d3ebaaea1680a8a566f80fe7d604843343374/amivapi/settings.py#L35